### PR TITLE
Add automation build script

### PR DESCRIPTION
**Pull request by nghiant2710**
cc @pcarranzav

### CODE REVIEW

File: automation/jenkins_build.sh [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+			extract_checksum 1 $NODE_VERSION "checksum"
+		;;
+		'armv7hf')
+			baseImage='resin/armv7hf-debian:jessie'
+			binaryArch='armv7hf'
+			binaryUrl=$resinUrl
+			extract_checksum 0 $NODE_VERSION "checksum"
+		;;
+		'armel')
+			baseImage='resin/armel-node:6.5-slim'
```
Comment (pcarranzav) [inline automation/jenkins_build.sh]: Shouldn't this be a debian like the others?

Comment (nghiant2710) [reply automation/jenkins_build.sh]: @pcarranzav the `armel-debian` image is not working atm so we will build on top of the old image

Comment (petrosagg) [reply automation/jenkins_build.sh]: @nghiant2710 what is not working there?

Comment (nghiant2710) [reply automation/jenkins_build.sh]: @petrosagg please this issue https://github.com/resin-io-library/base-images/issues/313. I've updated the PR to make the script only builds armel node v0.10.x image.

**Review by petrosagg — COMMENTED** [review]
No-op review to dismiss stale review request

**Pull request closed without merging**